man/tpm2_rsadecrypt.1.md

```diff
@@ -42,7 +42,7 @@ The key referenced by key-context is **required** to be:
 
     Output file path, record the decrypted data.
 
-  * **-s**, **--pad-scheme**=_SCHEME_:
+  * **-g**, **--halg**=_ALGORITHM_:
```

Comment:
it's not a hashing algorithm aka "halg". So something like --algorithm or --scheme would be better here.